sample-app/src/components/cards/StandardCard.tsx

```diff
@@ -41,47 +41,44 @@ interface CtaData {
   linkType: string
 }
 
+function isCtaData(data: unknown): data is CtaData {
+  if (typeof data !== 'object' || data === null) {
+    return false;
+  }
+  const ctaData = data as CtaData;
+  const expectedKeys = ['label', 'link', 'linkType'];
+  return expectedKeys.every(key => {
+    return key in ctaData;
+  });
```

Comment:
can we just do: 
```
const expectedKeys = ['label', 'link', 'linkType'];
  return expectedKeys.every(key => {
    return key in data;
  });
```
to directly check data without the force assertion?

Reply:
ya, that'll work. I tried that at first but there was an error because typescript couldn't guarantee the data was an object. I've since added a check to make sure it's an object, so the casting is no longer necessary